### ISSUE TITLE
YouTube has a new channel format not supported by Piped

### ISSUE THREAD

**Issue by Cade66** [TeamPiped/Piped#919]
Hi,

I was searching for ITV on DuckDuckGo and was surprised to find : `https://www.youtube.com/itv`
The url lands on `https://www.youtube.com/itv` : it is not modified by YouTube

Up to now we knew YouTube channels as being noted by `youtube.com/channel/`, `youtube.com/user/`, `youtube.com/c/`
It appears YouTube accepts -- at least for some channels -- the url : `youtube.com/[CHANNEL]`

Redirecting YouTube to Piped with the new format fails, Piped does not understand it and, i.e. `https://www.youtube.com/itv` lands on` https://piped.kavin.rocks/itv` and is not resolved.
But `https://piped.kavin.rocks/c/itv` is handled correctly (whilst **not** `https://piped.kavin.rocks/channel/itv`)

In other words, if not `channel/` nor` user/` nor `c/` nor `playlist`, the redirection requires, for Piped, the addition of `c/` at this time.

Brand new as far as I'm concerned, and bothering.

**Comment by arrowgent** [comment]
i dont think its too new, ive known it was an option when you can create a username
but its not the only method to access the username.

in youtube studio settings when you go to create your "channel name" this is where it is generated and accepts
youtube.com/username
youtube.com/c/username
etc

**Comment by Cade66** [comment]
I assumed perhaps too quickly that `youtube.com/username` was new on the ground it was the first time I encountered it.
It not being new is news here, so to say.
Personally and until Piped handles the format I've set a dedicated Redirector (extension) redirection to have `youtube.com/username` sent to `youtube.com/c/username`.

**Comment by FireMasterK** [comment]
Duplicate of #6
